sdk/communication/communication-phone-numbers/test/utils/recordedClient.ts

```diff
@@ -60,6 +60,15 @@ export function createRecordedClient(
   };
 }
 
+export const canCreateRecordedClientWithToken = (): boolean => {
```

Comment:
Nitpick / Suggestion. Instead of checking. 
```
if (!canCreateRecordedClientWithToken())
```
Could we not, in beforeEach
```
const recordedClient = createRecordedClientWithToken(this);
if (recordedClient) {
   ...
} else {
   this.skip();
}
```

Reply:
Yea that would work as well but that check would run for every test case. If I run in before it just checks once and skips the whole test suite if it can't create the client.